tensorflow/core/kernels/mkl/mkl_kernel_util.h

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_CORE_KERNELS_MKL_MKL_KERNEL_UTIL
+#define TENSORFLOW_CORE_KERNELS_MKL_MKL_KERNEL_UTIL
+
+#ifdef INTEL_MKL
+
+#include "tensorflow/core/common_runtime/kernel_benchmark_testlib.h"
+#include "tensorflow/core/framework/allocator.h"
+#include "tensorflow/core/framework/fake_input.h"
+#include "tensorflow/core/framework/node_def_builder.h"
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/framework/tensor_testutil.h"
+#include "tensorflow/core/framework/types.h"
+#include "tensorflow/core/framework/types.pb.h"
```

Comment:
Is this necessary? It doesn't seem to be used here?
We have a rule to not include proto headers in kernel files, so if it's necessary, we'll have to find a workaround, e.g., including some other header files that has this header, or move this util file outside of kernels/, etc.

Reply:
I removed the inclusion of this file, it was unnecessary. Thanks!